src/AdminConsole/Pages/App/Credentials/User.cshtml.cs

```diff
@@ -22,24 +21,16 @@ public class UserModel : PageModel
         ILogger<IndexModel> logger,
         IScopedPasswordlessClient api)
     {
-        Credentials = new CredentialsModel(authenticatorDataProvider);
         _logger = logger;
         _passwordlessClient = api;
     }
 
     public async Task OnGet()
     {
-        var items = await _passwordlessClient.ListCredentialsAsync(UserId);
-        Credentials.SetItems(items);
+        Credentials = await _passwordlessClient.ListCredentialsAsync(UserId);
         Aliases = await _passwordlessClient.ListAliasesAsync(UserId);
     }
 
-    public async Task<IActionResult> OnPost(string token)
-    {
-        var res = await _passwordlessClient.VerifyTokenAsync(token);
-        return new JsonResult(res);
-    }
-
```

Comment:
question: Why was this removed? Is it no longer needed?

Reply:
It was not used at all. Might have been a left-over from when the playground was developed. I'll double check again.

Reply:
@abergs No other references are found. Seems ok to go